test/runner.py

```diff
@@ -465,6 +468,11 @@ def prepend_default(arg):
     tests = tests_with_expanded_wildcards(tests, all_tests)
     tests = skip_requested_tests(tests, modules)
     tests = args_for_random_tests(tests, modules)
+
+  if not options.start_at and options._continue:
+    if os.path.exists(common.LAST_TEST):
+      options.start_at = utils.read_file(common.LAST_TEST).strip()
```

Comment:
Maybe error if there is no last test file?

Reply:
I think its better not too probably since then I can just run the same command ever time.. even for the first time.